### ISSUE TITLE
Confirm what’s going in the small, open workshop “room” to the left as you enter the Tapestry space

### ISSUE THREAD

**Issue by zarino** [DoESLiverpool/somebody-should#686]
Spun off from a comment by @ajlennon in #598:

---

PARTITION / POWER / NETWORK / SINK in this area

[This weekend] we were looking at this panel by the entrance to the main area. This is where it has been suggested we put a sink. Apparently plumbers think this might be a problem as the soil pipe is a way away by the kitchenette area.

So number one: Do we want a sink here? If not do we have two sinks in the kitchenette area (clean versus dirty as in Gostins?) Or something else?

Then number two: There don't seem to be any network points or power points here. If we put in network points and power points then we'd be set in future if we wanted to close off this space and make another meeting room. Now is the time to do it if we are going to do it. What are your thoughts?

If we really don't want to have power or networking points in this space then presumably the builders are free to finish up the partition wall here.

![image](https://user-images.githubusercontent.com/1537834/37283287-569bad02-25ef-11e8-95f9-c3dc61328fe2.png)

**Comment by zarino** [comment]
> Do we want a sink here? If not do we have two sinks in the kitchenette area (clean versus dirty as in Gostins?)

I can answer this one. The latest estimate we had to get water and waste over to that “room” was _way_ over what we could justify, and a few of us are starting to think it’d actually be handy to keep that room flexible, so it could potentially become another workspace area, if we start running out of space in the two enclosed workshop rooms.

Thus, I think we’re going to have the "dirty" sink in the kitchenette, here:

![screen shot 2018-03-12 at 12 45 30](https://user-images.githubusercontent.com/739624/37284537-c57dad66-25f3-11e8-88c8-4bcf00d3cc6d.png)

**Comment by ajlennon** [comment]
Great. That's what I understood from the discussion. So I'm going to take it as understood that the area referred to as the "sink area" is in fact "space" unless somebody wants to jump in and tell me it's wrong.

In which case onto the next question - shouldn't we have electric and networking points in this space now, whether or not in future we wish to turn it into another room?

**Comment by jackie1050** [comment]
It should. Most workshop equipment would require power.

**Comment by ajlennon** [comment]
On positioning of the dirty and clean sink

- i don't think we want the dirty area to be the first thing visitors to DoES see as they walk in.

- to use both of those surfaces for sinks seems a waste of space in the kitchenette and really would detract from its purpose imho

**Comment by zarino** [comment]
Yes, at the very least we’ll need power sockets in there (were they not on @johnmckerrell’s original diagram?) – I guess if we’re adding power sockets, we may as well have networking sockets in there too.

**Comment by ajlennon** [comment]
@jackie1050 is this something you could discuss with the spark?

**Comment by jackie1050** [comment]
Just spoken to @magman2112. He is expecting the electrician to come back today, and will discuss with him then.

Additionally @magman2112 is going to put network points in,as they will more than likely be needed for any future use (given that sink is unlikely to be here). He also suggested this area would be good for things like sewing machines  and sublimation printing, something to think about?

**Comment by johnmckerrell** [comment]
I did think sockets were going in on the opposite wall to this one but if we're quite sure a sink won't be going there it wouldn't hurt to add more (or even if the sink might go there but it won't interfere). TBH I'd then say "surely we want power *and* ethernet on both sides of that wall??"

Simon Derwent today said he didn't think a sink in this location was that much of an issue but it sounds like the decision has already been made.

If we say "workshop" and "food" sink does that sound less dirty? The workshop sink has tended to get less use anyway, and obviously everyone's going to tidy up after themselves when we get to our new shiny place, no??

**Comment by zarino** [comment]
Whenever I‘ve been into the current DoES kitchen, the "workshop" sink has always looked pretty tidy … Maybe I just happen to catch it on good days!!

That said, if we’re worried about having a sink/worktop on such prominent display as people come through the front door, then we could put both sinks side-by-side _inside_ the kitchen, and instead put something "tidier" like the fridge and a storage cupboard on that narrow wall:
 
![screen shot 2018-03-12 at 16 18 15](https://user-images.githubusercontent.com/739624/37296049-b55c6a58-2611-11e8-8b72-7b922f414ed2.png)

**Comment by johnmckerrell** [comment]
Where is the dishwasher going (hm.. maybe that's another ticket)?

**Comment by ajlennon** [comment]
That looks great to me @zarino !

**Comment by amcewen** [comment]
I'd flip which is which on the food/workshop labelling of those sinks, just because any workshop use that's messy would be coming from outside of that room and so would need to go _past_ the food sink to get to the workshop sink (and if nothing else, tempt people to just use the food sink...) but that's not really going to affect anything.

**Comment by zarino** [comment]
Yeah, the sinks could obviously be swapped 😉

**Comment by ajlennon** [comment]
I am hearing a lot about cleaning things yet strangely nothing about where the coffee will be made !?!

**Comment by jackie1050** [comment]
I don't really like the sinks being next to each other as a few people might be tempted to just use the first one they came across (ideally they should be in separate areas, but I can see that's causing difficulties...

On 12 Mar 2018 17:55, Zarino Zappia <notifications@github.com> wrote:

Yeah, the sinks could obviously be swapped 😉

**Comment by magman2112** [comment]
A possible practical point, any sinks will need to connect to the Soil pipe and I suspect that if the pipes need to go under the floor, big chunks would need to be cut from joists to get to this soil pipe. 

It might be better to come up with a few more potential layouts and then ask a plumber which are practical / cost effective.

**Comment by magman2112** [comment]
I can also confirm that we plan to put 4 network points in the open partition as shown on the photograph at the start of this thread, these should be installed tomorrow.

We may be a little late to put network points in the other wall, as this is planned to be boarded by the builders tomorrow. If I can get the cables in before the builders clad this wall I will do so. This wall already has a single twin socket installed towards the inner corner.

I did speak to John the electrician about adding another double socket in the open wall, but I don’t know if he has quoted a price to anyone as yet. He didn’t come back to me with a price.

**Comment by johnmckerrell** [comment]
FYI we've had a complaint about the noise of the fit-out from the ground floor tenants. They're being very nice about it but if you're working in the building please try to limit the noise you make between 10-3 Monday-Thursday and Saturday 9:30 - 4:30. Thanks!

**Comment by magman2112** [comment]
Power and network connections have now been run on both side walls of this open workshop space now, ahead of the builder boarding out the walls.

**Comment by johnmckerrell** [comment]
I don't dislike the idea of using this space for the vinyl cutter, printers, etc. but am wondering where we intend to put the storage that was going to go here? We're still pretty limited for storage space and this was about the only allocated area. I do support having that stuff out and available but as it's quiet equipment perhaps it could go on the wall next to the meeting room?

**Comment by johnmckerrell** [comment]
I've broken the kitchen/sinks out into another issue #690

**Comment by ajlennon** [comment]
Great news @magman2112 . I was wondering about the power on the outside of the door as presumably we need to power some door opening devices. Great you already have it there

**Comment by amcewen** [comment]
Just to throw some other "where does X go?" into the mix, we have still got a huge laser-cutter to put somewhere (once it's reassembled), and Martin has ordered a CNC router - not massive, but probably a bit bigger than Sophia.  Maybe we need to be thinking about how much room there is in the two workshop rooms...?

**Comment by magman2112** [comment]
One thing I was discussing with @johnmckerrell and @jackie1050 in recent days is the storage space we will need in Tapestry. The ex “Sink” space is one place that John was anticipating would be a primary storage space, especially for the very large shelves that have come across from Gostins.

We need to measure to be sure, but it looks like these shelves will fit on the back wall (towards the stairs) quite well. It may also be possible to stack these shelves on top of each other, given the height of the space available in Tapestry. Again, this needs to be measured, plus we would need some way to join the two sets of uprights vertically. Either we can find out if the manufacturer has joining connectors, or we can improvise them from large pieces of wood fitted into the uprights.

**Comment by ajlennon** [comment]
Referencing #718 should/could the laser cutter go here?

**Comment by jackie1050** [comment]
Maybe the larger laser cutter could be stored there temporarily, but if it's in use (as a laser cutter), it would need an exhaust.

On 25 Mar 2018 19:10, Alex Lennon <notifications@github.com> wrote:

Referencing #718<https://github.com/DoESLiverpool/somebody-should/issues/718> should/could the laser cutter go here?

**Comment by ajlennon** [comment]
Can't see it being in use there but it does have to go somewhere...

**Comment by amcewen** [comment]
We're still waiting for the skirting to be painted in here, but I think @jackie1050 is right, and the Monster laser-cutter needs to live there until it's operational, as there's not really anywhere else for it to go.

I _think_ there should also be space (if they're both stored end-on) in there for one of the sets of racking too, as there's no other obvious place for a second rack (if, as I mentioned in #674, the other one lives in the main room against the meeting room wall)
